styles/modules/modals/_settings.scss

```diff
@@ -14,14 +14,21 @@
     }
   }
 
+  .avatarPreview {
+    border: 0;
+
+    .cropit-preview-image-container {
+      border-radius: 60px;
```

Comment:
Cool, thanks. Never thought to use percentages on border-radius for some reason. Always thought it had to be pixels. No idea why. 🤔